src/content/saas-vs-self-hosted.mdx

```diff
@@ -0,0 +1,28 @@
+---
+title: SaaS vs. Self-Hosted
+description: The difference between Okteto's product offerings in SaaS and Self-Hosted
+sidebar_label: SaaS vs. Self-Hosted
+id: saas-vs-self-hosted
+---
+
+Okteto consists of two products: our [CLI](/cloud/okteto-cli) and the [Okteto Platform](#). You can use the Okteto CLI without using the Okteto platform, but there are important limitations with that use case. 
+
+The Okteto Platform is the other half of our product experience that provides the fully managed infrastructure for your development environments. It's what interprets the [Okteto Manifest](/reference/manifest) to spin up a development environment and automates your workflows. 
+
+Some examples of what the Okteto paltform manages are: sleeping/waking resources, incorporating external resources into your development environment, and automatically scaling your development environment as your application's needs grow.
+
+Within the context of the Okteto Platform we offer this experience in two ways: a fully-managed SaaS experience and a DIY self-hosted installation using a Helm chart.
+
+# Important things to know
+
+SaaS
+
+- Fully managed by Okteto
+- Limitations in configurations for complex environments
+- Easy and fast to get started
+
+Self-Hosted
+
+- More flexibility in the installation and configuration of the platform/experience
+- Fully managed by you as the customer
+- More complex to setup
```

Comment:
I'd like us to remove this, since this might send the wrong message

Reply:
Done